src/index.js

```diff
@@ -0,0 +1,8 @@
+import './style.css';
+import generateScore from './modules/generateScore.js';
+
+function component() {
+  generateScore('Abebe', 1);
+}
+
+component();
```

Comment:
- I suggest that this function listens to an event listener. You can consider using `DOMContentLoaded`. This ensures that you follow javascript best practices, please fix this 👍 👍